src/wrappers/google.js

```diff
@@ -135,7 +135,7 @@ function getEvents(jwtClient, calendarId) {
 
     return _.map(creatorEvents, function(event) {
       var startDate = moment(event.start.dateTime);
-      var niceStart = startDate.format('dddd MMMM Mo [at] hA'); // e.g., 'Monday February 2nd at 2PM'
+      var niceStart = startDate.format('dddd[,] MMMM Mo [at] hA'); // e.g., 'Monday February 2nd at 2PM'
```

Comment:
eventually we should create this in a filter that formats data instead of doing this inside of the google wrapper. this should just provide raw data.